scene/2d/cpu_particles_2d.cpp

```diff
@@ -537,6 +561,11 @@ void CPUParticles2D::_validate_property(PropertyInfo &p_property) const {
 	if (p_property.name.begins_with("scale_curve_") && !split_scale) {
 		p_property.usage = PROPERTY_USAGE_NONE;
 	}
+
+	if (p_property.name == "seed" && !use_fixed_seed) {
+		p_property.usage = PROPERTY_USAGE_NO_EDITOR;
+		return;
```

Comment:
It's the last condition, so return is redundant here.
I'd put it in the other conditions instead, or everywhere for consistency.